### ISSUE TITLE
Visual Studio Setting Page and Settings Menu Names

### ISSUE THREAD

**Issue by dtg3** [iTrace-Dev/iTrace-VisualStudio#33]
In the `tools->options` menu of iTraceVS the name should not be "iTrace" for the option and the settings page. It should match the tool and say `iTraceVS Plugin`. We changed this is Eclipse for consistency and it needs to match in the VS plugin was well.

**Comment by dtg3** [comment]
Once fixed the settings should be `iTraceVS Plugin` and the Settings page should say `iTraceVS Plugin Settings`. The port number setting should also say `iTraceVS Plugin Port Number`

**Comment by dtg3** [comment]
The name of the plugin was decided to be "iTrace VS Plugin". This is now complete and present in the latest version of the code base and installer.

NOTE: It might be necessary to remove profile settings data for VS. I noticed that in order to see the changes I had made to the interface and setting elements I had to remove my VS preferences in the AppData VS directories. I'm assuming this is just a by product of building and testing the plugin on the same VS environment.
